packages/ketchup/src/components/kup-input-panel/kup-input-panel.tsx

```diff
@@ -1989,36 +1990,39 @@ export class KupInputPanel {
         );
     }
 
-    #setFocusOnFirstInput(root:ShadowRoot){
-        
+    #setFocusOnFirstInput(root: ShadowRoot) {
         const form = root.querySelector('form');
-        const firstCellContent = form?.querySelector<HTMLElement>('.f-cell__content');
-    
+        const firstCellContent =
+            form?.querySelector<HTMLElement>('.f-cell__content');
```

Comment:
avoid using querySelector when possible (save the ref)

Reply:
taking form using formRef state. However, for firstCellContent there is another way to take 'f-cell__content' despite querySelector? formRef it's always an HTMLFormElement